src/pages/LoginPage.jsx

```diff
@@ -2,17 +2,30 @@ import React from 'react';
 import logo from '../logo.png';
 import Cookies from '../API/Cookies';
 import { Form, Input, Button } from 'antd';
+import PollService from '../API/PollService';
 
 function LoginPage({ setIsLogin }) {
   const onFinish = (values) => {
-    Cookies.setCookie('user-id', values.userId);
-    setIsLogin(Cookies.getCookie('user-id'));
+    const loginInfo = {
+      "email" : values.email,
+      "password": values.password
+    }
+    login(loginInfo)
   };
 
   const onFinishFailed = (errorInfo) => {
     console.log('Failed:', errorInfo);
   };
 
+  async function login(loginInfo) {
+    //setIsPollLoading(true);
+    const response = await PollService.login(loginInfo);
+    Cookies.setCookie('access-token', response['access-token']);
+    setIsLogin(Cookies.getCookie('access-token'));
```

Comment:
там же просто флажок true/false ожидается. давай его и поставим
```suggestion
    setIsLogin(true);
```